# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../acx100/configure.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- acx100-0.2.0pre8_plus_fixes_45/Configure	2004-11-23 09:00:58.000000000 +0000
+++ acx100-0.2.0pre8_plus_fixes_45-fixed/Configure	2005-02-21 02:06:18.869402736 +0000
@@ -9,12 +9,11 @@
 # some initial definitions
 
 CONFIG_MK=config.mk
-KERNEL_VER=`uname -r`
 KERNEL_BUILD=/lib/modules/$KERNEL_VER/build
 # If using 2.6 outputting to a seperate directory, make KERNEL_OUTPUT
 # point to that directory
 #KERNEL_OUTPUT=
-ECHO="`which echo` -e"
+ECHO="echo -e"
 WARNING_FLAG=0
 
 # functions
